packages/mobile/src/geth/networkConfig.ts

```diff
@@ -81,10 +81,10 @@ const FETCH_USER_LOCATION_DATA_PROD = `${CLOUD_FUNCTIONS_MAINNET}/fetchUserLocat
 const KOMENCI_LOAD_CHECK_ENDPOINT_STAGING = 'https://staging-komenci.azurefd.net/v1/ready'
 const KOMENCI_LOAD_CHECK_ENDPOINT_PROD = 'https://mainnet-komenci.azurefd.net/v1/ready'
 
-const SUPERCHARGE_AVAILABLE_REWARDS_URL_ALFAJORES =
-  'https://us-central1-celo-mobile-alfajores.cloudfunctions.net/fetchAvailableSuperchargeRewards'
-const SUPERCHARGE_AVAILABLE_REWARDS_URL_MAINNET =
-  'https://us-central1-celo-mobile-mainnet.cloudfunctions.net/fetchAvailableSuperchargeRewards'
+const CLOUD_FUNCTIONS_BASE_URL_ALFAJORES =
+  'https://us-central1-celo-mobile-alfajores.cloudfunctions.net'
+const CLOUD_FUNCTIONS_BASE_URL_MAINNET =
+  'https://us-central1-celo-mobile-mainnet.cloudfunctions.net'
```

Comment:
We already have definitions for these in https://github.com/valora-inc/wallet/blob/f0ce9f8ff359941aa615f5593cf70b2c1805ff88/packages/mobile/src/geth/networkConfig.ts#L51-L52